dash/frontend/src/app/modules/private/pages/cluster/kubernetes-namespaces/kubernetes-namespaces.component.html

```diff
@@ -1,5 +1,5 @@
-<div class="page-container">
-  <div class="h-xs-fit page-container-content">
+<div class="row display-xs-block h-xs-100 overflow-xs-auto mr-xs-1">
```

Comment:
is
`mr-xs-1`
the same as
`me-xs-1`?

Reply:
@becky-intelletive  yes. we have just updated mr-xs-1 to me-xs-1.